saleor/graphql/payment/tests/test_utils.py

```diff
@@ -0,0 +1,288 @@
+from unittest.mock import patch
+
+import pytest
+from django.core.exceptions import ValidationError
+
+from ....page.models import PageType
+from ....payment.error_codes import (
+    TransactionRequestActionErrorCode,
+    TransactionRequestRefundForGrantedRefundErrorCode,
+)
+from ....site.models import SiteSettings
+from ..utils import validate_and_resolve_refund_reason_context
+
+
+@patch("saleor.graphql.payment.utils.Site.objects.get_current")
+def test_no_reference_type_configured_no_reference_id_provided(mock_get_current):
```

Comment:
Mock is not needed at all. 
```python
def test_no_reference_type_configured_no_reference_id_provided(site_settings):
    # Given
    site_settings.refund_reason_reference_type = None
    site_settings.save()
```

IMHO it should be changed to all tests in this file. It would be better to use existing site_settings instead of mocking